docs/reference/commandline/service_update.md

```diff
@@ -21,52 +21,52 @@ Usage:  docker service update [OPTIONS] SERVICE
+      --container-label-add list           Add or update a container label (default [])
+      --container-label-rm list            Remove a container label by its key (default [])
+      --endpoint-mode string               Endpoint mode (vip or dnsrr)
+      --env-add list                       Add or update an environment variable (default [])
+      --env-rm list                        Remove an environment variable (default [])
+      --force                              Force update even if no changes require it
+      --group-add stringSlice              Add an additional supplementary user group to the container
+      --group-rm list                      Remove a previously added supplementary user group from the container (default [])
+      --health-cmd string                  Command to run to check health
+      --health-interval duration-ptr       Time between running the check (default none)
+      --health-retries int                 Consecutive failures needed to report unhealthy
+      --health-timeout duration-ptr        Maximum time to allow one check to run (default none)
+      --help                               Print usage
+      --image string                       Service image tag
+      --label-add list                     Add or update a service label (default [])
+      --label-rm list                      Remove a label by its key (default [])
+      --limit-cpu NanoCPUs                 Limit CPUs (default 0.000)
+      --limit-memory MemoryBytes           Limit Memory (default 0 B)
+      --log-driver string                  Logging driver for service
+      --log-opt list                       Logging driver options (default [])
+      --mount-add mount                    Add or update a mount on a service
+      --mount-rm list                      Remove a mount by its target path (default [])
+      --no-healthcheck                     Disable any container-specified HEALTHCHECK
+      --publish-add list                   Add or update a published port (default [])
+      --publish-rm list                    Remove a published port by its target port (default [])
+      --replicas uint64-ptr                Number of tasks (default none)
+      --reserve-cpu NanoCPUs               Reserve CPUs (default 0.000)
+      --reserve-memory MemoryBytes         Reserve Memory (default 0 B)
+      --restart-condition string           Restart when condition is met (none, on-failure, or any)
+      --restart-delay duration-ptr         Delay between restart attempts (default none)
+      --restart-max-attempts uint64-ptr    Maximum number of restarts before giving up (default none)
+      --restart-window duration-ptr        Window used to evaluate the restart policy (default none)
+      --rollback                           Rollback to previous specification
+      --stop-grace-period duration-ptr     Time to wait before force killing a container (default none)
```

Comment:
Not for this PR, but we should change the output to be a bit more useful; `duration-ptr` is the internal implementation, and not very informative for the end-user.